syntax = "proto3";

import "Task.proto";
import "TaskId.proto";

message TaskToSerialize
{
  TaskId id = 1;
  Task task = 2;
  TaskId parent_id = 3;
}

message TaskPackage
{
  repeated TaskToSerialize tasks = 1;
}